### ISSUE TITLE
Incorrect classNames typedef

### ISSUE THREAD

**Issue by chasemcdo** [react-tags/react-tags#979]
Hey team!

I am looking to use this package in one of my projects, but am having trouble applying classes to it. I am following the documentation and have something like:
```javascript
<ReactTags
  classNames={{
    root: "bg-red-500",
    rootFocused: "bg-red-500",
    selected: "bg-red-500",
    selectedTag: "bg-red-500",
    selectedTagName: "bg-red-500",
    search: "bg-red-500",
    searchInput: "bg-red-500",
    suggestions: "bg-red-500",
    suggestionActive: "bg-red-500",
    suggestionDisabled: "bg-red-500",
  }}
/>
```

Where the Tailwind Class `bg-red-500` is being used as an obvious placeholder for relevant classes; however, the only classes that seem to impact the appearance are those applied to `selected` and on the `Tag` objects themselves. Am I simply misunderstanding how these should be impacting my UI?

To further elaborate, if I try to add any other attributes to classNames, I'll receive type errors as I am running in a typed node environment, but it seems the classNames type def is missing relevant ones while containing unused ones:

https://github.com/react-tags/react-tags/blob/cb3c1c073034be1e93a7a0d35d9707a25bf6e1ea/src/index.tsx#L143-L154

Thanks!

**Comment by chasemcdo** [comment]
Created a [small PR](https://github.com/react-tags/react-tags/pull/980/files), which is likely missing some classes, but updates it to align with the constants definitions here:

https://github.com/react-tags/react-tags/blob/cb3c1c073034be1e93a7a0d35d9707a25bf6e1ea/src/components/constants.ts#L24-L36

**Comment by chasemcdo** [comment]
Workaround for anyone else hitting a similar issue is to simply typecast the classNames input as any or you can type cast it as the correct version from above.

ie
```typescript
<ReactTags
  classNames={{
    selected: "bg-red-500",
    tagInputField: "rounded-lg text-sm px-2 py-1",
  } as any}
/>
```

**Comment by ad1992** [comment]
closed by #980
